langlib/lang.__internal/src/main/java/org/ballerinalang/langlib/internal/GetElementType.java

```diff
@@ -42,16 +43,23 @@ public static BTypedesc getElementType(Object td) {
     }
 
     private static BTypedesc getTypeDescValue(Type type) {
```

Comment:
Shall we rename the method name as `getElementTypeDescValue`. ref:https://github.com/ballerina-platform/ballerina-lang/pull/32248#discussion_r692872819

Reply:
I will do this.